docs/usage/plotting.rst

```diff
@@ -0,0 +1,40 @@
+.. _usage-plotting:
+
+Plotting
+========
+
+Boost-histogram does not contain plotting functions - this is outside of the
+scope of the project, which is histogram filling and manipulation. However, it
+does follow ``PlottableProtocol``, as listed below. Any plotting library that
+accepts an object that follows the ``PlottableProtocol`` can plot boost-histogram
+objects.
+
+Using the protocol:
+
+Plotters should only depend on the methods and attributes listed below. In short, they are:
+
+* ``h.kind``: The bh.Kind of the histogram (COUNT or MEAN)
+* ``h.values()``: The value (as given by the kind)
+* ``h.variances()``: The variance in the value (None if an unweighed histogram was filled with weights)
+* ``h.counts()``: The effective counts
```

Comment:
Please change to "How many values the bin received or their effective number if the histogram is weighted".

Reply:
I am open to the exact wording, but you cannot explain .counts() with the word counts.

Reply:
Also, there is no such thing as an effective count for an unweighted histogram.